webapp-django/crashstats/crashstats/forms.py

```diff
@@ -235,6 +235,16 @@ def clean_v(self):
             )
         return versions
 
+    def clean(self):
+        cleaned_data = super(DailyFormBase, self).clean()
+        if cleaned_data.get('date_end') and cleaned_data.get('date_start'):
+            # Check the invariant. Make sure the start < end.
+            if cleaned_data['date_start'] > cleaned_data['date_end']:
+                raise forms.ValidationError(
+                    'date start > date end'
```

Comment:
I'll rewrite it to say "Start date greater than end date". I think that's more useful to whoever has to see this.